packages/@adobe/spectrum-css-temp/components/barloader/index.css

```diff
@@ -22,6 +22,10 @@ governing permissions and limitations under the License.
   inline-size: var(--spectrum-barloader-large-width);
   vertical-align: top;
   isolation: isolate;
+
+  &.spectrum-BarLoader--indeterminate {
+    max-width: var(--spectrum-barloader-large-width);
+  }
```

Comment:
Where did this come from?

Reply:
Indeterminate progress bars are locked to 192px until we change the animations to work with custom widths.

Reply:
gotcha, that should fine then